plugins/catalog-import/src/components/StepFinishImportLocation/StepFinishImportLocation.tsx

```diff
@@ -46,7 +46,12 @@ const filterComponentEntity = (
           entity.kind.toLocaleLowerCase('en-US'),
         )
       ) {
-        return entity;
+        return {
+          kind: entity.kind.toLocaleLowerCase('en-US'),
+          namespace:
+            entity.namespace?.toLocaleLowerCase('en-US') ?? DEFAULT_NAMESPACE,
+          name: entity.name,
```

Comment:
just to double check whether `entity.name` deserves the same massaging

Reply:
For now, I think `entity.name` do not fix the same way. Because of `catalogPage` define "kind element mapper" using the lower case of kind
<img width="485" alt="image" src="https://github.com/backstage/backstage/assets/7720722/c4744eff-ca7d-410c-8191-1ef9890d1426">